Common/cpp/SharedItems/Shareables.h

```diff
@@ -223,6 +224,28 @@ class ShareableHostFunction : public Shareable {
   unsigned int paramCount_;
 };
 
+class ShareableArrayBuffer : public Shareable {
+ public:
+  ShareableArrayBuffer(
+      jsi::Runtime &rt,
+#if REACT_NATIVE_MINOR_VERSION >= 72
+      const jsi::ArrayBuffer &arrayBuffer
+#else
+      jsi::ArrayBuffer arrayBuffer
+#endif
+      )
+      : Shareable(ArrayBufferType),
+        data_(
+            arrayBuffer.data(rt),
+            arrayBuffer.data(rt) + arrayBuffer.size(rt)) {
+  }
+
+  jsi::Value toJSValue(jsi::Runtime &rt) override;
+
+ protected:
+  const std::vector<uint8_t> data_;
```

Comment:
Could you confirm that `uint8_t` works as well for example with `Float64Array`?

Reply:
`uint8_t` is often used to represent a byte so effectively `std::vector<uint8_t>` is just an array of bytes. We never access or modify the data from C++ side, we just treat it as a binary buffer. Theoretically, it should work, but I will confirm it by adding more examples as you suggested in the next comment.

Reply:
Thank you 😊 I also think that is should work, but it is better to check to be sure.